lib/mongodb-wrapper.js

```diff
@@ -82,13 +86,15 @@ module.exports.MongoWrapper = class MongoWrapper {
 				try {
 					clients[dbConfigKey] = await this.connect(config);
 
-					Events.once('janiscommerce.ended', this.closeConnections);
+					if(this.closeConnection !== 'false')
```

Comment:
Sabes que se me hace muy raro que valides con un `string` que contiene `'false'`.
Estuve buscando en internet para ver si las variables de entorno se podían settear como booleanas, pero no encontré, capaz que no ajajaj o capaz busqué mal. 

Pero te dejo mi duda ( no es una corrección ).

Reply:
Yo también estuve buscando, y según lo que encontré es que una vez seteada la variable de entorno, cuando se accede a ella es siempre un `string`, por eso no importa el tipo de dato que mandes (en mi caso en los tests probé enviando `boolean` o `string` y cumple con la validación.
Te dejo dos links de referencia:
[Link 1](https://github.com/node-config/node-config/issues/272)
[Link 2](https://stackoverflow.com/questions/59599304/proper-way-to-parse-environment-variables)